lang_qc/models/pacbio/well.py

```diff
@@ -132,9 +133,10 @@ def pre_root(cls, values: dict[str, Any]) -> dict[str, Any]:
         """
 
         # https://github.com/pydantic/pydantic-core/blob/main/python/pydantic_core/_pydantic_core.pyi
-        mlwh_db_row: PacBioRunWellMetrics = values.kwargs["db_well"]
-        assert mlwh_db_row
+        if "db_well" not in values.kwargs:
```

Comment:
The type hint says values is a dict. Does this work with just `if "db_well" not in values:`?

Reply:
Also it seems strange that Pydantic can't just make `db_well` a required attribute without you having to check it.

Reply:
`values` seems to be of type [ArgsKwarg ](https://github.com/pydantic/pydantic-core/blob/66331bb1a4977d3d927fd26c6929dd6532d586ef/python/pydantic_core/_pydantic_core.pyi#L864), which does not seem to be a part of public API.